_posts/2019-14-11-designup-bangalore.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: event
+title:  "Open Design @ DesignUP!: Bangalore, Nov 14th"
+date:   2019-11-14
+categories: design hack meeting
+eventDate: Thursday, 14th Nov 2019
+location: Bangalore, India
+time: full days
+status: future
```

Comment:
This can be set to "past" now I guess?

@AnXh3L0 @ei8fdb @dmichl do you know if Jekyll could automatically know based on the date whether it’s future or past? Cause adjusting this always is kind of annoying, no?

Reply:
@jancborchardt My knowledge of Jekyll is ➡️this big⬅️, i.e. tiny. From a search on jekyll dates past and future, [I found a GH issue](https://github.com/jekyll/jekyll/issues/3882).

I'm not fully sure. :/